lib/webchat.yaml

```diff
@@ -1,12 +1,12 @@
 - base_path: /government/organisations/hm-passport-office/contact/passport-advice-and-complaints
-  open_url: https://hmpowebchat.klick2contact.com/v03/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://hmpowebchat.klick2contact.com/v03&u=&wo=&uh=&pid=2&iif=0
-  availability_url: https://hmpowebchat.klick2contact.com/v03/providers/HMPO/api/availability.php
+  open_url: https://omni.eckoh.uk/v03.5/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://omni.eckoh.uk/v03.5&u=&wo=&uh=&pid=2&iif=0
+  availability_url: https://omni.eckoh.uk/v03.5/providers/HMPO/api/availability.php
   open_url_redirect: false
 - base_path: /government/organisations/hm-passport-office/contact/webchat
-  open_url: https://hmpowebchat.klick2contact.com/v03/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://hmpowebchat.klick2contact.com/v03&u=&wo=&uh=&pid=2&iif=0
-  availability_url: https://hmpowebchat.klick2contact.com/v03/providers/HMPO/api/availability.php
+  open_url: https://omni.eckoh.uk/v03.5/launcherV3.php?p=HMPO&d=717&ch=CH&psk=chat_a1&iid=STC&srbp=0&fcl=0&r=Chat&s=https://omni.eckoh.uk/v03.5&u=&wo=&uh=&pid=2&iif=0
+  availability_url: https://omni.eckoh.uk/v03.5/providers/HMPO/api/availability.php
   open_url_redirect: false
```

Comment:
This [page](https://www.gov.uk/government/organisations/hm-passport-office/contact/webchat) doesn't exist, so I think this entry can be removed as well.

Reply:
@leenagupte Both links refer to https://www.gov.uk/government/organisations/hm-passport-office/contact/hm-passport-office-webchat which very much exists and is in use.

Reply:
Screw it. Removed both. Fix incoming.

Reply:
@Tetrino This page: https://www.gov.uk/government/organisations/hm-passport-office/contact/webchat 404s.

Reply:
@leenagupte All done.

Reply:
Have you been able to test that it works?

Reply:
I tried it locally and got "Invalid CSRF token"

Reply:
@leenagupte You will do, because of security rules that are handled in https://github.com/alphagov/govuk_app_config/pull/225/files - in the draft version of this, there was a temporary override that allowed the requester to test it all on a heroku machine. It's been working fine for about three weeks on that machine.